backend/src/routes/auto_trader.py

```
@@ -27,4 +27,27 @@ async def stop_auto_trader():
+        logger.error(f"Error fetching auto trader config: {str(e)}")
+        return error_response('Failed to fetch auto trader configuration', details=e)
+
+@auto_trader_bp.route('/config', methods=['POST'])
+def update_auto_trader_config():
+    """Updates the auto trader configuration."""
+    auto_trader_service = current_app.services['auto_trader']
+    try:
+        new_config = request.get_json()
+        auto_trader_service.update_config(new_config)
+        return success_response(message='Auto trader configuration updated successfully.')
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)